src/Squisher.js

```diff
@@ -52,23 +55,79 @@ class Squisher {
             }
         }
 
-        this.ids = new Set();
-        this.entities = new Array();
-        this.clickListeners = new Array(this.width * this.height);
         this.update(this.root);
+        // acts as a heartbeat
```

Comment:
Will make this comment more descriptive. I noticed that in Chrome, if there was no activity on the socket for a few seconds, the next frame was delayed a bit. Keeping the socket alive with the current frame every now and then fixes that issue